integration-tests/api/medusa-config.js

```diff
@@ -14,6 +15,7 @@ module.exports = {
     database_type: "postgres",
     jwt_secret: "test",
     cookie_secret: "test",
+    response_compression_enabled: enableResponseCompression
```

Comment:
suggestion:Should we move the option to `response_compression_options` which could have `enable` true or false?